.github/workflows/refapp-3x-settings.yml

```diff
@@ -14,7 +14,14 @@ jobs:
       run:
         working-directory: ./qaframework-bdd-tests
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
```

Comment:
There's no need to add this section to the workflow because `repository: ${{github.repository}}` contains a default value.

Reply:
Sure, should I remove it?